src/vmm/src/device_manager/persist.rs

```diff
@@ -173,9 +166,9 @@ pub struct ConnectedLegacyState {
 }
 
 /// Holds the MMDS data store version.
-// NOTICE: Any changes to this structure require a snapshot version bump.
-#[derive(Debug, Clone, PartialEq, Eq, Versionize)]
+#[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub enum MmdsVersionState {
+    #[default]
```

Comment:
maybe `V2` is a more sensible default?

Reply:
In fact, we should completely remove the `Default` implementation here. The state will always be stored in the snapshot file. It doesn't make sense to have this at all, since we will never trigger it.